ru/turing/courses/lesson2/Homework2.java

```diff
@@ -0,0 +1,105 @@
+package ru.turing.courses.lesson2;
+
+
+import ru.turing.courses.lesson2.animals.Animal;
+import ru.turing.courses.lesson2.animals.Cat;
+import ru.turing.courses.lesson2.animals.Mouse;
+
+import java.util.Calendar;
+import java.util.GregorianCalendar;
+import java.util.Scanner;
+
+public class Homework2 {
+    public static void main(String[] args) {
+
+        //Животные
+        Mouse mouse = new Mouse("Мышыч", "серая", "basic", 10, 4, "Бри", "8");
+        mouse.sleep(mouse.getHoursOfSleep());
+        mouse.krodetsya();
+        mouse.eat(mouse.getFavoriteTypeOfCheese());
+
+        Cat cat = new Cat("Барсик", "Коричневый", "Британский", 5, 4, "10", "Whiskas");
+        cat.play(cat.getPlayWithLaser());
+        cat.eat(cat.getFavoriteTypeOfFeed());
+        cat.cheel();
+
+        //подключаю библиотеку и создаю переменную с текущим годом
+        Calendar calendar = new GregorianCalendar();
+        int currentYear = calendar.get(Calendar.YEAR);
+
+        //ввод фио пользователя
+        System.out.println("Введите свое ФИО: ");
+        Scanner scan = new Scanner(System.in);
+        String fio = scan.nextLine();
+
+
+        //считаю количество полных лет и месяцев пользователя
+
+        int age = countAge(currentYear);
+
+        System.out.println("Количество полных лет ползователя: " + age);
+
+
+        System.out.println("Количество полных месяцев ползователя: " + countMonth(age));
+
+        //вызываю функцию перевода адресса
+        shorAdressFormat();
+
+    }
+
+    public static void shorAdressFormat() {
+        Scanner scan = new Scanner(System.in);
+
+        //ввод пользователем адресса
+        System.out.println("Введите свой адресс одной строкой (Пример - страна: Россия, город: Москва, улица: Авиамоторная, дом: 15, квартира: 24)");
+        //создаю символ который в дальнейшем буду удалаять из нужной мне перепенной
+        char charToRemove = ',';
+
+        //для каждой переменной создаю переменну-буфер, в которой хранится страна:, город: и т.п.
+        String stranaBuffer = scan.next();
+        String strana = scan.next();
+        //удаляю запятую. Без этой строчки вывод переменной strana выглядел бы так -> Россия,
+        strana = strana.replace(String.valueOf(charToRemove), "");
+
+        //Повторяю те же действия для каждой составляющей адресса
+        String gorodBuffer = scan.next();
```

Comment:
```suggestion
        String cityBuffer = scan.next();
```
И дальше тоже исправь